repo-index.yml

```diff
@@ -512,7 +512,7 @@ plugins:
   version: 1.0.0
   created: 2015-11-11
   updated: 2015-11-11
-  company: 
+  company:
```

Comment:
that wasn't an intentional change -- i would guess atom automatically removed that space